### PR TITLE
DOCS(readme): Mention SignPath

### LINKED ISSUE #6128: Mention about SignPath Foundation and free code certificate in Readme.md missing
### Description

Hi
I am the Developer Evangelist from SignPath.

I noticed that there is no mention of us in the readme. 
Could you please add this to your Readme:

```
## Code Signing
This program uses free code signing provided by [SignPath.io](https://signpath.io?utm_source=foundation&utm_medium=github&utm_campaign=mumble), and a free code signing certificate by the [SignPath Foundation](https://signpath.org?utm_source=foundation&utm_medium=github&utm_campaign=mumble)
```

If you want, I can provide a Pull Request for this change.

Thanks!

### Steps to reproduce

-

### Mumble version

Source

### Mumble component

Both

### OS

Other

### Reproducible?

Yes

### Additional information

_No response_

### Relevant log output

_No response_

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by Krzmbrzl**
Fixes #6128

### Checks

- [x] My commits follow the [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,12 @@ If you want to report a bug or create a feature-request, you can open a new issu
 [GitHub](https://github.com/mumble-voip/mumble/issues/new/choose).
 
 
+# Code Signing
```
Comment (ppfeufer) [inline README.md:59]: This should be a `##` (`h2` headline), not `#` (`h1`, or main headline) as it breaks the headline semantics.

An `h1` should be the main headline and only be used once in a document.

As it is currently, it feels that `Windows`, `MacOS` and everything following are sub-parts of `Code Signing`.

image

Comment (ppfeufer) [reply README.md:59]: A generated Table of Contents from the current document structure shows this pretty clearly:

image

Comment (Krzmbrzl) [reply README.md:59]: Indeed - that's what you get for copy-pasting xD

Comment (Krzmbrzl) [reply README.md]: Thanks for bringing this up!

**Pull request merged**